src/js/angular/security/controllers.js

```diff
@@ -104,6 +104,37 @@ const parseAuthorities = function (authorities) {
     };
 };
 
+const compactAuthorities = function (authorities) {
```

Comment:
It's hard to understand what this function does without reading the code and why. Can you drop some doc comment here. Other than that, I wonder, why this is not placed inside some service?